services/api.ts

```diff
@@ -14,3 +14,43 @@ export const get = async <T>(url: string, params: ParamsType): Promise<{
     statusCode: response.status,
   };
 }
+
+export const post = async <T>(url: string, data: T): Promise<{
+  data: T,
+  statusCode: number,
+}> => {
+  const response = await axios.post(url, data);
```

Comment:
write to `const response = await axios.post<T>(url, data);`

Reply:
done